src/main/java/commands/CreateTask.java

```diff
@@ -0,0 +1,42 @@
+package commands;
+
+import myTasks.Deadline;
+import myTasks.Event;
+import myTasks.Task;
+import myTasks.Todo;
+
+import java.util.List;
+
+/**
+ * The CreateTask class contain methods to create new tasks for the user to track.
+ */
+public class CreateTask {
+    /**
+     * The method todo creates a task without restrictions.
+     * @param description contains the description of the task.
+     * @param list contains the list of tasks that are currently being tracked.
+     */
+    public static void todo(String description, List<Task> list) {
```

Comment:
Perhaps a more meaningful name could be used instead of `list`, such as:
- `todoList`
- `taskList`
- etc